oddbird/sass-site
# Debug links

## Description
In the debug console, line numbers are now links to select or position the cursor at the relevant issue. Also, deprecation messages with links are now clickable.

## Related Issue(s)
Addresses #75

## Steps to test/reproduce

1. [Debug, Warn, and Deprecation links](https://deploy-preview-76--sass-site-oddbird.netlify.app/playground/#eJwzNHRISU0qTVfQMDQqqNA3KajQtOYCAEPTBfY=?s=L1C9-L1C17)
2.[ Error link](https://deploy-preview-76--sass-site-oddbird.netlify.app/playground/#eJwzNFRJz89PsVJISkyx5uJKrOZSUEjOz8kvslJQKUoFCtUCAKdZCcA=?s=L4C10-L4C14)
3. [Error without a range](https://deploy-preview-76--sass-site-oddbird.netlify.app/playground/#eJwzNExJTUkqTVcAAA4tAu4=)

Links should allow you to open in new tab/window with CMD/Shift/Right click, etc.
